src/Microsoft.TestPlatform.Common/Microsoft.TestPlatform.Common.csproj

```diff
@@ -26,6 +26,11 @@
       <FromP2P>true</FromP2P>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.4' ">
```

Comment:
Please add a comment to maintain context of this change. Same comment everywhere.

Which API in XmlReaderWriter is not available for UAP?

Reply:
it's not the API, but since we have taken dependency on version 4.3.0, it files while trying to load xmlreader, as the one present in UWP is 4.0.11 by default.